integrations/monsterapi.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: integration
+name: MonsterAPI
+description: Use open Language Models served by MonsterAPI
+authors:
+    - name: monsterapi
+      socials:
+        github: qblocks
+        twitter: monsterapi
+        linkedin: monsterapi
```

Comment:
Can you provide the full URL for the LinkedIn account? e.g. for deepset it is: https://www.linkedin.com/company/deepset-ai

Reply:
> Can you provide the full URL for the LinkedIn account? e.g. for deepset it is: https://www.linkedin.com/company/deepset-ai


Sure. As requested, we have changed the link for LinkedIn in the new commit.